server/events.lua

```diff
@@ -168,18 +168,17 @@ RegisterNetEvent('QBCore:Server:UseItem', function(item)
     QBCore.Functions.UseItem(src, item)
 end)
 
+-- This event is exploitable and should not be used. Will be deprecated soon.
 RegisterNetEvent('QBCore:Server:RemoveItem', function(itemName, amount, slot)
     local src = source
     local Player = QBCore.Functions.GetPlayer(src)
     if not Player then return end
     Player.Functions.RemoveItem(itemName, amount, slot)
```

Comment:
Add the print here as well like you did for AddItem but make it an error instead